packages/mds-api-server/index.ts

```diff
@@ -31,7 +31,7 @@ export type ApiQuery<Q1 extends string, Q2 extends string[] = never> = {
 
 export interface ApiResponse<L = unknown, B = unknown>
   extends express.Response<
-    B | { error: unknown; error_description?: string; error_details?: string[] } | { errors: unknown[] } | {}
+    B | { error: unknown; error_description?: string; error_details?: string[] } | { errors: unknown[] }
```

Comment:
Would it make sense to make this it's own interface
```ts
interface ApiErrorReponse {
  error: unknown
  error_description?: string
  error_details?: string[] 
}
```
And make the AprResponse `<B | ApiErrorResponse | { errors: ApiErrorReponse[] }`